### PR TITLE
chore: Upgrade ejs to fix async security alert from ejs 

### LINKED ISSUE #392: Noticed security issue async
Saw the nested alert with the vulnerability to `async`. https://github.com/advisories/GHSA-fwr7-v2mv-hh25 

Let me know if any one else is looking to potentially update before spelunking 

___ 

Looks like the relevant issue may be with ejs and potentially unneeded dep https://github.com/mde/ejs/issues/659

### PR DESCRIPTION
**Pull request by JackHowa**
- Fixes #392 and addresses one https://github.com/advisories/GHSA-fwr7-v2mv-hh25
- Thanks for the time @jondot @fyyyyy 
- This comes from a dependency update from jake and ejs https://github.com/mde/ejs/commit/c028c343c127859f7189c3feee1e5239c199fec9 https://github.com/jakejs/jake/pull/411 

# Testing 
- `yarn test` works and passing

### CODE REVIEW

**Review by fyyyyy — APPROVED** [review]

**Comment by JackHowa** [comment]
@fyyyyy @jondot is there a way that I can get approval to run the workflow? thanks so much for your time on this; really appreciate it. 

Screen Shot 2022-04-21 at 09 58 31

**Comment by fyyyyy** [comment]
I am not a maintainer of this project, only jondot

**Comment by JackHowa** [comment]
@jondot are you available to work on this project? I really appreciate your work on this and the flexibility that hygen gives us. Let me know if there's something I can help with to address this

**Comment by JackHowa** [comment]
@jondot let me know if this project is not going to support the security update. I can close the issue and pr -- no worries. I get it that oss is hard to support and continue

**Comment by JackHowa** [comment]
Will close this as will not fix. Let me know if this is mistaken @jondot @fyyyyy

**Pull request closed without merging**
